Tools/WebKitTestRunner/TestController.cpp

```diff
@@ -2770,7 +2780,12 @@ void TestController::decidePolicyForUserMediaPermissionRequestIfPossible()
         auto& settings = settingsForOrigin(originHash);
         settings.incrementRequestCount();
 
-        if (!m_isUserMediaPermissionAllowed && !settings.persistentPermission()) {
```

Comment:
Is persistent permission no longer supported?

Reply:
persistentPermission is limited to enumerateDevices showing deviceIds in that case.
We should deprecate that code path since the spec mandates exposing deviceId once getUserMedia is successfully called.